pkg/cmd/ctlstore/main.go

```diff
@@ -275,10 +285,24 @@ func supervisor(ctx context.Context, args []string) {
 		if cliCfg.Shadow {
 			shadow = "true"
 		}
+
+		var promHandler *prometheus.Handler
```

Comment:
Seems like you're configuring prometheus for the supervisor here, not the reflector.  The supervisor does have a reflector but I imagine you want to add this to the `reflector` func.

Reply:
you're right :) fixed !